src/doc/help.py

```diff
@@ -339,3 +341,17 @@ def help_attribute_table():
     ]
     html = html_table(title, intro, inputs, outputs, more)
     return html
+#ajoute par Rachel Goree le 30/05/2017
```

Comment:
It is not necessary to keep this line if it can be noted that the development was realized by Rachel Gorée.